.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+## Pull Request
+
+### Description
```

Comment:
As most of the PRs will be related to the change of the components, can we include in  PR the template a URL to the storybook to see and test changes quickly?

Reply:
Not sure about this, but if it's added, in the case some new change is committed, the link would be outdated, since a new build is done after each commit

Reply:
I will add an extra instruction in the testing steps